src/MakeCommand.php

```diff
@@ -96,6 +96,17 @@ public function execute(InputInterface $input, OutputInterface $output)
     }
 
     /**
+     * Slugifies the Project Name.
+     *
+     * @param  string $projectName
```

Comment:
@pluxain missing space between `string` and `$projectName`, they must be two of them.